scripts/posteffects/posteffect-bloom.js

```diff
@@ -184,27 +184,33 @@ function BloomEffect(graphicsDevice) {
 BloomEffect.prototype = Object.create(pc.PostEffect.prototype);
 BloomEffect.prototype.constructor = BloomEffect;
 
+BloomEffect.prototype._destroy = function () {
+    if (this.targets) {
+        var i;
+        for (i = 0; i < this.targets.length; i++) {
+            this.targets[i].destroyTextureBuffers();
```

Comment:
was it intentional to no longer call `destroyFrameBuffers` ?

Reply:
Great, now I'm wondering why destroy doesn't also do `destroyTextureBuffers()`. @mvaligursky any idea?

Reply:
Yes. Framebuffers we allocate internally, we deallocate them.
Color buffers are allocated by the users, they need to deallocate. Part of the reason here is that a single color buffer can be used for multiple render targets, and we dont want to ref count them to handle release properly.

Reply:
I had the same thought :) Probably because a render target can own the textures, but can also alias them. In the case of an aliasing situation, you would destroy the textures even if they're still in use.

Reply:
makes sense - thanks!